apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -207,11 +336,17 @@ class SbbolFintechApi extends SbbolRequestApi {
             method: 'GET',
             path: this.getPaymentRequestStatePath.replace(':externalId', externalId),
         })
-        const jsonData = JSON.parse(data)
-        if (statusCode === 200) {
-            return { data: jsonData }
-        } else {
-            return { error: jsonData }
+
+        try {
+            const jsonData = JSON.parse(data)
+
+            if (statusCode === 200) {
+                return { data: jsonData }
+            } else {
+                return { error: jsonData }
```

Comment:
statusCode ?

Reply:
it seems that the statusCode is not needed for the logic further